docs/observability/aws/integrations/amazon-sqs.md

```diff
@@ -0,0 +1,165 @@
+---
+id: amazon-sqs
```

Comment:
Hi @JKashyap96 , is [this](https://help.sumologic.com/docs/integrations/amazon-aws/sqs/) the same app?

Reply:
HI @kimsauce ! This is the same app but we maintain a different page here as a few sections are not included in this page from the one you mentioned.